Content/Items/Misc/MartialBook.cs

```diff
@@ -0,0 +1,43 @@
+﻿using StarlightRiver.Content.Items.BaseTypes;
+using System.Collections.Generic;
+using Terraria.ID;
+
+namespace StarlightRiver.Content.Items.Misc
+{
+	public class MartialBook : SmartAccessory
+	{
+		public override string Texture => AssetDirectory.MiscItem + Name;
+
+		public MartialBook() : base("Martial Encyclopedia", "Combined effects of Tiger, Snake, and Mantis Techniques\n+10% melee critical damage\nQuit, don't quit... Noodles, don't noodles.") { }
+
+		public override List<int> ChildTypes =>
+			new()
+			{
+				ModContent.ItemType<SwordBook>(),
+				ModContent.ItemType<SpearBook>(),
+				ModContent.ItemType<AxeBook>()
+			};
+
+		public override void SafeUpdateEquip(Player Player)
+		{
+			Player.GetCritChance(DamageClass.Melee) += 10;
```

Comment:
This is inconsistent with the tooltip. This adds 10% crit -chance- while the tooltip says 10% crit -damage-